proposals/validators.py

```diff
@@ -1,21 +1,33 @@
 from django.forms import forms
 from django.utils.translation import gettext as _
 
-from .models import Proposal
 
 
 class UniqueTitleValidator:
+    
+    
     proposal = None
 
-    def __init__(self, proposal: Proposal = None):
+    def __init__(self, proposal = None):
         self.proposal = proposal
 
     def __call__(self, value):
+        
+        # Importing here to prevent circular import
+        from .models import Proposal
+        
         qs = Proposal.objects.filter(title=value)
 
         if self.proposal:
             qs = qs.exclude(pk=self.proposal.pk)
 
         if qs.exists():
             raise forms.ValidationError(_('Er bestaat al een studie met deze '
-                                          'titel.'), code='unique')
\ No newline at end of file
+                                          'titel.'), code='unique')
+
+def AVGUnderstoodValidator(value):
```

Comment:
Any reason to use a whole validator over just marking the DB field as required?

Reply:
So far, this is the only way I've found to work around both #126 and #123 (very much pet peeves of mine) in the current proposal form. Either you can't navigate away from the page, or you use SoftValidationMixin and accept that an invalid value will never be saved because of a DB constraint.

The latter admittedly only comes in to play with a user checking the box, saving the form, and then later trying to uncheck it, which is kind of their own fault for trying to un-read something. But this feels like the kind of question where there should be no possibility of the user "unintentionally" having this box checked.